packages/form-core/src/validate/resultValidators/DefaultSuccess.js

```diff
@@ -1,14 +1,29 @@
 import { ResultValidator } from '../ResultValidator.js';
 
+/**
+ * @typedef {import('../Validator').Validator} Validator
+ */
+
 export class DefaultSuccess extends ResultValidator {
+  /**
+   * @param  {...any} args
+   */
   constructor(...args) {
     super(...args);
     this.type = 'success';
   }
 
+  /**
+   *
+   * @param {Object} context
```

Comment:
what is actually the difference between `Object` and `object`? 
Do we use the habit of lowercasing platform primitives (like we do with string and boolean?)

Reply:
I think me and @narzac  agreed we should uppercase non-primitives as a convention, although I'm sure you can find some lowercased `function` type annotations that I made a mistake in. To JSDocs it makes 0 difference so it's just a convention we can choose.